### PR TITLE
Added the quackathon/corrected date for royalhackaway

### PR DESCRIPTION
**Pull request by ksouthern**

### CODE REVIEW

**Comment by robzwolf** [comment]
There are conflicts here, probably because of #15... Can you fix them?

Also – their website is https://quackathon.com/

**Comment by bahorn** [comment]
Fixed the merge issue

File: docs/index.md [hunk 1]
```diff
@@ -32,9 +32,10 @@ This list showcases upcoming UK student-run hackathons.
 | Oxford Hack 2019 | [oxfordhack.co.uk](https://oxfordhack.co.uk/) | Oxford University | 300 | 16th-17th November 2019 |
 | GreatUniHack 2019 | [greatunihack.com](https://greatunihack.com/) | University of Manchester | 200 | 16th-17th November 2019 |
 | DurHack 2019    | [durhack.com](https://durhack.com) | Durham University | 150 | 23rd-24th November 2019 |
-| ManMetHacks 2.0 | [ManMetHacks](https://manmethacks.com) | Manchester Metropolitan University | 150 | 25th-26th January 2020 |
+| ManMetHacks 2.0 | [ManMetHacks](https://manmethacks.com) | Manchester Metropolitan University | 150 | 25th-26th January 2019|
```
Comment (robzwolf) [inline docs/index.md]: This should be 2020 – merge gone wrong? @bahorn

File: docs/index.md [hunk 2]
```diff
@@ -32,9 +32,10 @@ This list showcases upcoming UK student-run hackathons.
...
+| Royal Hackaway v3 | [royalackaway.com](https://royalhackaway.com/) | Royal Holloway, University of London | 300 | 1st-2nd February 2020|
```
Comment (robzwolf) [inline docs/index.md]: Missing `h` in `royalackaway.com`

File: docs/index.md [hunk 3]
```diff
@@ -32,9 +32,10 @@ This list showcases upcoming UK student-run hackathons.
...
+| Quackathon 2020 | [quackathon.com](https://quackathon.com/) | University of Dundee | 100 | 1st-2nd February 2020|
```
Comment (robzwolf) [inline docs/index.md]: 👍

File: docs/index.md (hunk 1 above)
Comment (bahorn) [reply docs/index.md]: Yep, think so.

**Comment by robzwolf** [comment]
Would argue it’s still not ready for merging, need to fix the ManMetHacks date and the typo in the Royal Hackaway URL @bahorn @ksouthern

**Comment by bahorn** [comment]
Yeah, that was me making a stupid mistake.

File: docs/index.md [hunk 4]
```diff
@@ -32,9 +32,10 @@ This list showcases upcoming UK student-run hackathons.
 | Oxford Hack 2019 | [oxfordhack.co.uk](https://oxfordhack.co.uk/) | Oxford University | 300 | 16th-17th November 2019 |
 | GreatUniHack 2019 | [greatunihack.com](https://greatunihack.com/) | University of Manchester | 200 | 16th-17th November 2019 |
 | DurHack 2019    | [durhack.com](https://durhack.com) | Durham University | 150 | 23rd-24th November 2019 |
-| ManMetHacks 2.0 | [ManMetHacks](https://manmethacks.com) | Manchester Metropolitan University | 150 | 25th-26th January 2020 |
+| ManMetHacks 2.0 | [ManMetHacks](https://manmethacks.com) | Manchester Metropolitan University | 150 | 25th-26th January 2020
```
Comment (robzwolf) [inline docs/index.md]: Technically this line should end in a `|` @ksouthern 😛

**Pull request merged**